fcrepo-import/src/main/java/org/fcrepo/importer/Importer.java

```diff
@@ -83,14 +87,10 @@ private FcrepoClient client() {
      */
     public void run() {
         logger.info("Running importer...");
-        try {
```

Comment:
Why are you removing the exception handling here and converting the Exceptions below to RuntimeExceptions?  I think it's important to make sure the Exceptions get logged and don't just get passed up to the command line.